_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -838,6 +835,27 @@ public function can_request( $request ) {
 			return current_user_can( 'jetpack_configure_modules' );
 		}
 	}
+
+	/**
+	 * Allows the `settings` and `module/<slug>` EDITABLE endpoints to accept both JSON and multi-part POST bodies.
+	 *
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return array|bool
+	 */
+	public function parse_settings_request_body( $request ) {
```

Comment:
What problem does this function solve?  In general, API endpoints should pull the parameters they need directly from `$request['parameter_name']`, which has the benefits of simplicity and of treating parameters equally from all sources.  From #5973:

> almost all endpoints should access parameters directly like `$request['parameter_name']` so that they can come from any source and still be treated correctly.  I can't think of a reason why an endpoint would still need to get all its parameters at once, but in that case it should use `$request->get_params()` to get all the parameters from every source, rather than only accepting JSON.

Given that, and given that this function is only called once that I see, I would recommend removing it.